devsecops/security-scanning/xebialabs/README.md.tmpl

```diff
@@ -0,0 +1,47 @@
+{{$app := .AppName | kebabcase}}
+## Simple security and compliance template and dashboard
+
+Security and compliance are critical aspects of product release. Organizations struggle to track application release status information and understand security and compliance risks across many different applications, teams, and environments. When risk assessment, security testing, and compliance checks aren’t built into the Continuous Integration/Continuous Delivery (CI/CD) pipeline, releases fail and cause delays, security vulnerabilities threaten production, and IT governance violations result in expensive fines.
+
+This blueprint provides a way to integrate tools like Sonarqube, Fortify, Checkmarx, and Black duck into release pipeline.
+
+#### Sonarqube
```

Comment:
Just a suggestion, maybe it would be better to include those parts of the readme with if and elses since you already ask user to use fortify/checkmarx/blackduck or not